### PR TITLE
[Merged by Bors] - systest: add test that forces smeshers to cast neutral votes

### LINKED ISSUE #3159: systest: implement test that prevents hare progress and triggers round of neutral votes
based on the observations from latest testnets any problems in hare create a set of cascading issues and network doesn't make any progress. the goal of the test is to make hare run for max number of iterations. i expect that it will be achievable by one of the chaos scenarios:

- adjusting time on part of nodes by 5-10s
- disconnecting part of nodes

### PR DESCRIPTION
**Pull request by dshulyak**
closes: https://github.com/spacemeshos/go-spacemesh/issues/3159

### CODE REVIEW

**Comment by dshulyak** [comment]
bors try

File: systest/tests/timeskew_test.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+		skewOffset = "-3s" // hare round is 2s
+	)
+	tctx := testcontext.New(t, testcontext.Labels("sanity"))
+	cl, err := cluster.Default(tctx, cluster.WithKeys(10))
+	require.NoError(t, err)
+
+	failed := int(0.2 * float64(tctx.ClusterSize))
+	eg, ctx := errgroup.WithContext(tctx)
+	client := cl.Client(0)
+	scheduleChaos(ctx, eg, client, enableSkew, stopSkew, func(ctx context.Context) (chaos.Teardown, error) {
```
Comment (countvonzero) [inline systest/tests/timeskew_test.go:30]: can you add comments roughly about what this function `chaos.Timeskew` do?

**Review by countvonzero — APPROVED** [review]

File: systest/tests/timeskew_test.go (hunk 1 above)
Comment (countvonzero) [reply systest/tests/timeskew_test.go:30]: i mean how does this function cause hare to have the two failure scenarios below?

Comment (dshulyak) [reply systest/tests/timeskew_test.go:30]: there was a doc about chaos.Timeskew, i updated a comment in the test as well

**Comment by dshulyak** [comment]
bors merge

File: systest/tests/timeskew_test.go (hunk 1 above)
Comment (countvonzero) [reply systest/tests/timeskew_test.go:30]: is it possible that 2 hare committees are formed, one from the 80% and one from the 20% skewed? don't know hare that well...

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors merge

File: systest/tests/timeskew_test.go (hunk 1 above)
Comment (dshulyak) [reply systest/tests/timeskew_test.go:30]: i think this is possible if they received different sets of activations, e.g. there was a long network partition

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors merge

**Pull request closed without merging**